src/main/java/xyz/groundx/caver_ext_kas/kas/KAS.java

```diff
@@ -75,14 +76,15 @@ public KAS initAnchorAPI(String chainId, String accessKeyId, String secretAccess
      * @param accessKeyId The access key provided by KAS console.
      * @param secretAccessKey The secret key provided by KAS console.
      * @param url An URL to request Wallet API.
+     * @param rpc The RPC for using Node API.
      */
-    public KAS initWalletAPI(String chainId, String accessKeyId, String secretAccessKey, String url) {
+    public KAS initWalletAPI(String chainId, String accessKeyId, String secretAccessKey, String url, RPC rpc) {
```

Comment:
How about use a setter method for setting a RPC instance?

Reply:
@sirano11 
I applied your feed back [here](https://github.com/ground-x/caver-java-ext-kas/pull/80/commits/70c84ac2ad6e1f04c82281ea63363fbbf95eab3a) 🙂